JASP-Engine/JASP/R/ttestindependentsamples.R

```diff
@@ -1,698 +1,695 @@
-			colnames(summaryStat)[which(colnames(summaryStat) == .v(options$groupingVariable))] <- "groupingVariable"
-			
-			pd <- ggplot2::position_dodge(.2)
-						
-			p <- ggplot2::ggplot(summaryStat, ggplot2::aes(x=groupingVariable, y=dependent, group=1)) +
-				 ggplot2::geom_errorbar(ggplot2::aes(ymin=ciLower, ymax=ciUpper), colour="black", width=.2, position=pd) +
-				 ggplot2::geom_line(position=pd, size = .7) + 
-				 ggplot2::geom_point(position=pd, size=4) +
-				 ggplot2::ylab(unlist(options$variables[var])) +
-				 ggplot2::xlab(options$groupingVariable) +
-				 ggplot2::theme_bw() +
-				 ggplot2::ggtitle(options$variables[var]) +
-				 ggplot2::theme(panel.grid.minor=ggplot2::element_blank(), plot.title = ggplot2::element_text(size=18),
-					panel.grid.major=ggplot2::element_blank(),
-					axis.title.x = ggplot2::element_text(size=18,vjust=-.2), axis.title.y = ggplot2::element_text(size=18,vjust=-1),
-					axis.text.x = ggplot2::element_text(size=15), axis.text.y = ggplot2::element_text(size=15),
-					panel.background = ggplot2::element_rect(fill = 'transparent', colour = NA),
-					plot.background = ggplot2::element_rect(fill = 'transparent', colour = NA),
-					legend.background = ggplot2::element_rect(fill = 'transparent', colour = NA),
-					panel.border = ggplot2::element_blank(), axis.line = ggplot2::element_blank(),
-					legend.key = ggplot2::element_blank(),
-					legend.title = ggplot2::element_text(size=12),
-					legend.text = ggplot2::element_text(size = 12),
-					axis.ticks = ggplot2::element_line(size = 0.5),
-					axis.ticks.margin = grid::unit(1,"mm"),
-					axis.ticks.length = grid::unit(3, "mm"),
-					plot.margin = grid::unit(c(.5,0,.5,.5), "cm")) +
-				 base_breaks_y(summaryStat) +
-				 base_breaks_x(summaryStat$groupingVariable)
-								
-			image <- .beginSaveImage(options$plotWidth, options$plotHeight)
-			print(p)
-			content <- .endSaveImage(image)
-
-			descriptivesPlot[["data"]] <- content
-
-			descriptivesPlotList[[var]] <- descriptivesPlot
-			
-		}
-		
-	} else {
+.ttestIndependentSamplesNormalityTest <- function(dataset, options, perform) {
+    if (options$normalityTests == FALSE) return(NULL)
+    
+    normalityTests <- list("title" = "Test of Normality (Shapiro-Wilk)")
+    
+    ## these are the table fields associated with the normality test
+    fields <- list(
+        list(name = "dep", type = "string", title = "", combine = TRUE), 
+        list(name = "lev", type = "string", title = ""),
+        list(name = "W", title = "W", type = "number", format = "sf:4;dp:3"),
+        list(name = "p", title = "p", type = "number", format = "dp:3;p:.001")
+    )
+    
+    normalityTests[["schema"]] <- list(fields = fields)
+    
+    footnotes <- .newFootnotes()
+    .addFootnote(footnotes, symbol = "<em>Note.</em>",
+                 text = "Significant results indicate a deviation from normality")
```

Comment:
We prefer milder language for significance; we go for *suggest* rather than *indicate*
and i think language about *violating an assumption* is more consistent with what we've done so far (not completely sure).

`Significant results suggest a violation of the assumption of normality`